consultation_analyser/support_console/ingest.py

```diff
@@ -57,150 +56,145 @@ def get_themefinder_outputs_for_question(
     return json.loads(response["Body"].read())
 
 
-def import_themes(question_part: QuestionPart, theme_data: dict) -> Framework:
-    theme_generation_execution_run = ExecutionRun.objects.create(
-        type=ExecutionRun.TaskType.THEME_GENERATION
-    )
-    framework = Framework.create_initial_framework(
-        question_part=question_part, execution_run=theme_generation_execution_run
-    )
-    for theme_key, theme_value in theme_data.items():
-        name, description = theme_value.split(": ", 1)
-        logger.info(f"Creating theme: {name}, key: {theme_key}")
-        Theme.create_initial_theme(
-            framework=framework, key=theme_key, name=name, description=description
+def import_respondent_data(consultation: Consultation, respondent_data: list):
+    logger.info(f"Importing respondent data for consultation: {consultation.title}")
+    respondents_to_save = []
+    for respondent in respondent_data:
+        respondent = json.loads(respondent.decode("utf-8"))
+        themefinder_respondent_id = respondent["themefinder_id"]
+        # TODO - add further fields e.g. user supplied ID
```

Comment:
just to check - is this data we don't have yet that we're planning to add later?